From 400febc88695bbab538a9aaa7f0af35a89a31d58 Mon Sep 17 00:00:00 2001
From: Rob Herring <r.herring@freescale.com>
Date: Fri, 6 Feb 2009 16:43:29 -0600
Subject: [PATCH 038/256] ENGR00107392 Move asm/arch headers to linux inc dir

Architecture specific headers moved and are not accessible to userspace in
2.6.28. So they are moved to linux include directory.

Signed-off-by: Rob Herring <r.herring@freescale.com>
---
 ipu/mxc_ipu_lib.c          | 4 ++--
 sahara2/fsl_shw_keystore.c | 6 +-----
 2 files changed, 3 insertions(+), 7 deletions(-)

diff --git a/ipu/mxc_ipu_lib.c b/ipu/mxc_ipu_lib.c
index ed8d186..53ceff3 100644
--- a/ipu/mxc_ipu_lib.c
+++ b/ipu/mxc_ipu_lib.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2004-2009 Freescale Semiconductor, Inc. All Rights Reserved.
  *
  */
 
@@ -34,7 +34,7 @@
 #include <sys/mman.h>
 #include <sys/ioctl.h>
 #include <sys/types.h>
-#include <asm/arch/ipu.h>
+#include <linux/ipu.h>
 
 static int fd_ipu = -1;
 static int open_count = 0;
diff --git a/sahara2/fsl_shw_keystore.c b/sahara2/fsl_shw_keystore.c
index 8b3c325..ea5f7e0 100644
--- a/sahara2/fsl_shw_keystore.c
+++ b/sahara2/fsl_shw_keystore.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2008-2009 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -25,10 +25,6 @@
 #include <diagnostic.h>
 #endif
 
-#if !defined(FSL_HAVE_SCC2) && defined(__KERNEL__)
-#include <asm/arch/mxc_scc_driver.h>
-#endif
-
 /* Define a semaphore to protect the keystore data */
 #ifdef __KERNEL__
 #define LOCK_INCLUDES os_lock_context_t context
-- 
1.8.0

